src/cluster/dcommands/admin/autoresponse.ts

```diff
@@ -125,13 +125,13 @@ export class AutoResponseCommand extends GuildCommand {
         const responseBase = `The raw code for ${match.id === 'everything' ? 'the everything autoresponse' : `autoresponse ${match.id}`} is`;
 
         const response = this.success(`${responseBase}:\n${codeBlock(match.ar.content)}`);
-        return guard.checkMessageSize(response)
+        return !match.ar.content.includes('`') && guard.checkMessageSize(response)
```

Comment:
why do you check for no backticks in the content?

Reply:
Because backticks inside a codeblock might break the codeblock, and if we add escapes then the codeblock technically no longer shows the raw of the tag. Easier and less error prone to just send the file in that case